doc/howto/network_load_balancers.md

```diff
@@ -78,14 +78,14 @@ Property          | Type       | Required | Description
 :--               | :--        | :--      | :--
 name              | string     | yes      | Name of the backend
 target\_address   | string     | yes      | IP address to forward to
-target\_port      | string     | no       | Target port(s) (e.g. `70,80-90` or `90`), same as `listen_port` if empty
+target\_port      | string     | no       | Target port(s) (e.g. `70,80-90` or `90`), same as the {ref}`port <network-load-balancers-port-specifications>`'s `listen_port` if empty
 description       | string     | no       | Description of backend
 
 (network-load-balancers-port-specifications)=
 ## Configure ports
 
-You can add port specifications to the network load balancer to forward traffic from specific ports on the listen address to specific ports one one or more target backends.
-It must be within the same subnet as the network that the load balancer is associated to.
+You can add port specifications to the network load balancer to forward traffic from specific ports on the listen address to specific ports on one or more target backends.
+The port's listen address must be within the same subnet as the network that the load balancer is associated to.
```

Comment:
Is that what we mean with "it" here?

Reply:
I think the entire sentence `It must be within the same subnet as the network that the load balancer is associated to.` acn be dropped.

A port doesn't have a listen_address, only the load-balancer does.

Reply:
@tomponline

Reply:
Yep agreed.

Reply:
That statement only relevant for backends